Formula/mrboom.rb

```diff
@@ -0,0 +1,19 @@
+class Mrboom < Formula
+  desc "Mr.Boom is an 8 player Bomberman clone"
+  homepage "http://mrboom.mumblecore.org"
+  url "https://github.com/Javanaise/mrboom-libretro/archive/3.8.tar.gz"
+  sha256 "2c05a68f0d513013e28934c3ef8f3b95f5330790ca2c8bd05a5413c3d5edd087"
+
+  depends_on "cmake" => :build
+  depends_on "libmodplug"
+  depends_on "lzlib"
+  depends_on "minizip"
+  depends_on "sdl2"
+  depends_on "sdl2_mixer"
+
+  def install
+    system "make", "mrboom", "LIBSDL2=1"
```

Comment:
`mrboom` is already the default target

Reply:
I don't think this line is needed at all, if you move the "LIBSDL2=1" down to be part of the `make install` line.

Reply:
no it's not, default is the libretro library, you need to run "make mrboom LIBSDL2=1" for the SDL2 version

Reply:
ah ok